packages/cms-cli/commands/customObject/schema/associations/create.js

```diff
@@ -42,9 +43,9 @@ exports.handler = async options => {
   try {
     // const res = await createSchema(portalId, filePath);
     // logger.success(
-    //   `Schema can be viewed at ${
+    //   `Schema can be viewed at ${getHubSpotWebsiteOrigin(
     //     getEnv() === 'qa' ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD
```

Comment:
Minor, but I noticed that when `qa` is added, this the path is incorrect (it would be `api.hubspotqa.com` or similar)

Reply:
Scratch that, I see that this was fixed!